form/multiple_buttons.rst

```diff
@@ -31,3 +31,12 @@ querying if the "Save and add" button was clicked::
 
         return $this->redirectToRoute($nextAction);
     }
+    
+Or you can get the button's name by using the 
+:method:`Symfony\\Component\\Form\\Form::getClickedButton` method of the form::
+    
+    if ('saveAndAdd' === $form->getClickedButton()->getName()) {
```

Comment:
We need to be careful here. Depending on how the form was submitted `getClickedButton()` may return `null`. Then, this example would run into an error as `getName()` could not be executed.